lua/wikis/heroes/GetMatchGroupCopyPaste/wiki.lua

```diff
@@ -0,0 +1,99 @@
+---
+-- @Liquipedia
+-- wiki=heroes
+-- page=Module:GetMatchGroupCopyPaste/wiki
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Class = require('Module:Class')
+local Logic = require('Module:Logic')
+local Lua = require('Module:Lua')
+
+local BaseCopyPaste = Lua.import('Module:GetMatchGroupCopyPaste/wiki/Base')
+
+---@class HeroesMatch2CopyPaste: Match2CopyPasteBase
+local WikiCopyPaste = Class.new(BaseCopyPaste)
+
+local INDENT = WikiCopyPaste.Indent
+
+--returns the Code for a Match, depending on the input
+---@param bestof integer
+---@param mode string
+---@param index integer
+---@param opponents integer
+---@param args table
+---@return string
+function WikiCopyPaste.getMatchCode(bestof, mode, index, opponents, args)
+	local showScore = Logic.nilOr(Logic.readBool(args.score), true)
+	local bans = Logic.readBool(args.bans)
+	local veto = Logic.readBool(args.veto)
+	local vetoBanRounds = tonumber(args.vetoBanRounds) or 0
+
+	local lines = Array.extend(
+		'{{Match',
+		bestof ~= 0 and (INDENT .. '|bestof=' .. bestof) or nil,
+		Logic.readBool(args.needsWinner) and INDENT .. '|winner=' or nil,
+		Logic.readBool(args.hasDate) and {
+			INDENT .. '|date=',
+			INDENT .. '|twitch=',
+			INDENT .. '|caster1=',
+			INDENT .. '|caster2=',
+			INDENT .. '|comment='
```

Comment:
How are these connected to `date`?

Reply:
I think casterX could be separate into a separate call, I assume that back then when I tried set it up I had no clue on how to make a separate caller for toggling caster inputs so I wrap it into date part 

Providing the form doesnt have a boolean for casters that's probably why 
![image](https://github.com/user-attachments/assets/b0c92f3a-6a33-4e3b-98f9-4031711af8b7)